calendar-amqp/src/test/java/com/linagora/calendar/amqp/AlarmEventUpdateTest.java

```diff
@@ -456,57 +456,6 @@ void shouldUpdateAlarmTimeOfAttendeeWhenOrganizerChangeStartDateLater() {
         });
     }
 
-    @Test
-    void shouldUpdateIcsPayloadOfAttendeeWhenOrganizerChangeEvent() {
```

Comment:
This test is legit and tests https://github.com/linagora/esn-sabre/issues/227

IMO let's not remove it.

Let's mark it @Disabled and reference the ticket